src/content/app/tools/blast/components/blast-job-submit/BlastJobSubmit.tsx

```diff
@@ -15,17 +15,39 @@
  */
 
 import React from 'react';
-import noop from 'lodash/noop';
+import { useSelector } from 'react-redux';
 
 import { PrimaryButton } from 'src/shared/components/button/Button';
 
+import { getBlastFormData } from '../../state/blast-form/blastFormSelectors';
+import { BlastFormState } from '../../state/blast-form/blastFormSlice';
+
 const BlastJobSubmit = () => {
   // TODO:
   // 1) validate the blast form before enabling the button
   // 2) actually do the job submission
+  const blastFormData = useSelector(getBlastFormData);
+
+  const onBlastSubmit = () => {
+    const postData = createBlastSubmissionData(blastFormData);
+    return postData;
+  };
+
+  const createBlastSubmissionData = (blastFormData: BlastFormState) => {
+    return {
+      email: 'blast2020@ebi.ac.uk',
+      genomeIds: blastFormData.selectedSpecies,
+      program: blastFormData.settings.program,
```

Comment:
shouldn't this be part of the parameters?

Reply:
I always thought the main configs that needs the tool to run are on the top level and all other configs go inside `parameters`  (`state` resembles this?)